python/kwiver/vital/types/camera_perspective.cxx

```diff
@@ -0,0 +1,271 @@
+/*ckwg +29
+ * Copyright 2017, 2020 by Kitware, Inc.
```

Comment:
For consistency use 2017-2020, but this looks like a new file which should be just 2020.